semgrep-core/tests/tainting_rules/go/token_labels.yaml

```diff
@@ -0,0 +1,16 @@
+rules:
+  - id: go-token-labels
+    message: A taint label did not prevent the match coming from a tainted token! 
```

Comment:
I got confused so maybe specify that the point is this is an unsatisfiable label and we're testing that the label works. Having the issue helps though